client/src/featureform/cli.py

```diff
@@ -155,6 +157,31 @@ def version():
     print(output)
 
 
+@cli.command()
+@click.option(
+    "--host",
+    "host",
+    required=False,
+    help="The host address for the arrow flight client to connect to",
+)
+@click.option(
+    "--cert", "cert", required=False, help="Path to self-signed TLS certificate"
+)
+@click.option("--insecure", is_flag=True, help="Disables TLS verification")
+@click.argument("name", required=True)
+@click.argument("variant", required=True)
+def stream(host, cert, insecure, name, variant):
```

Comment:
rename this "head" and give it a limit number, since I imagine this would be used to sample a dataset/resource